api-reference/20 Data Visualization Widgets/dxPieChart/1 Configuration/annotations/annotations.md

```diff
@@ -5,7 +5,207 @@ inherits: dxPieChart.Options.commonAnnotationSettings
+
+    @NgModule({
+        declarations: [
+            AppComponent
+        ],
+        imports: [
+            BrowserModule,
+            DxPieChartModule
+        ],
+        providers: [ ],
+        bootstrap: [AppComponent]
+    })
+    export class AppModule { }
+
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxPieChart ... >
+            <DxAnnotation
+                type="text"
+                text="Annotation text"
+            />
+            <DxAnnotation
+                type="image"
+                image="http://image/url/myimage.png"
+            />
+            <DxAnnotation
+                type="custom"
+                template="custom-annotation"
+            />
+            <template #custom-annotation="{ data }">
+                <svg>
+                    <!-- Declare custom SVG markup here -->
+                </svg>
+            </template>
+        </DxPieChart>
+    </template>
+
+    <script>
+    import DxPieChart, {
+        DxAnnotation
+    } from 'devextreme-vue/pie-chart';
+
+    export default {
+        components: {
+            DxPieChart,
+            DxAnnotation
+        },
+        data() {
+            // ...
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import PieChart, {
+        Annotation
+    } from 'devextreme-react/pie-chart';
+
+    function CustomAnnotation(annotation) {
+        const data = annotation.data;
+        return (
+            <svg>
+                {/* Declare custom SVG markup here */}
+            </svg>
+        );
+    }
+
+    export default function App() {
+        return (
+            <PieChart ... >
+                <Annotation
+                    type="text"
+                    text="Annotation text"
+                />
+                <Annotation
+                    type="image"
+                    image="http://image/url/myimage.png"
+                />
+                <Annotation
+                    type="custom"
+                    render={CustomAnnotation}
+                />
+            </PieChart>
+        );
+    }
+
+---
+
+Annotations can be anchored to a **PieChart** element. To do this, use the [argument]({basewidgetpath}/Configuration/annotations/#argument) or [series]({basewidgetpath}/Configuration/annotations/#series) options, depending on the element you want to anchor the annotation to.
+
+To define the position of an unanchored annotation, set the pixel coordinates ([x]({basewidgetpath}/Configuration/annotations/#x) and [y]({basewidgetpath}/Configuration/annotations/#y)) options.
+
+- **Unanchored annotation**
+
+        annotations: [{
+            x: 100,
+            y: 200
+        }]
+
+- **Annotation anchored to a series or series point**
+
+        annotations: [{
+            argument: "California",
+            series: "States"
+        }]
+
+- **Annotation anchored to an argument's edge or center**
```

Comment:
Since annotations are positioned at the center by default, I suggest demonstrating only how to position them at the edge.

```suggestion
- **Annotation positioned at an argument's edge**
```